press/docker/Dockerfile-Bench-Update

```diff
@@ -0,0 +1,173 @@
+# syntax = docker/dockerfile:experimental
+FROM ubuntu:20.04
+
+{% for d in doc.dependencies %}
+ENV {{ d.dependency }} {{ d.version }}
+{% else %}
+ENV NVM_VERSION 0.36.0
+ENV NODE_VERSION 14.19.0
+ENV PYTHON_VERSION 3.10
+ENV WKHTMLTOPDF_VERSION 0.12.5
+ENV BENCH_VERSION 5.15.2
+{% endfor %}
+
+ENV LANG C.UTF-8
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV OPENBLAS_NUM_THREADS 1
+ENV MKL_NUM_THREADS 1
+
+# Install essential packages
+RUN --mount=type=cache,target=/var/cache/apt apt-get update \
+  && apt-get install --yes --no-install-suggests --no-install-recommends \
+  # Essentials
+  build-essential \
+  git \
+  mariadb-client \
+  libmariadb-dev \
+  pv \
+  ntp \
+  wget \
+  supervisor \
+  # Dependencies for SSH access
+  openssh-server \
+  nano \
+  vim \
+  less \
+  htop \
+  # Dependencies for adding Python PPA
+  software-properties-common \
+  gnupg \
+  # weasyprint dependencies
+  libpango-1.0-0 \
+  libharfbuzz0b \
+  libpangoft2-1.0-0 \
+  libpangocairo-1.0-0 \
+  # wkhtmltopdf dependencies
+  ca-certificates \
+  fontconfig \
+  libfreetype6 \
+  libjpeg-turbo8 \
+  libpng16-16 \
+  libx11-6 \
+  libxcb1 \
+  libxext6 \
+  libxrender1 \
+  xfonts-75dpi \
+  xfonts-base \
+  # pycups dependencies
+  gcc \
+  libcups2-dev \
+  # s3-attachment dependencies
+  libmagic1 \
+  && rm -rf /var/lib/apt/lists/* \
+  `#stage-pre-essentials`
+
+COPY --chown=root:root supervisord.conf /etc/supervisor/supervisord.conf
+
+# Install Redis from PPA
+RUN --mount=type=cache,target=/var/cache/apt add-apt-repository ppa:redislabs/redis \
+  && apt-get update \
+  && apt-get install --yes --no-install-suggests --no-install-recommends \
+  redis-server \
+  && rm -rf /var/lib/apt/lists/* `#stage-pre-redis`
+
+# Install Python from DeadSnakes PPA
+RUN --mount=type=cache,target=/var/cache/apt add-apt-repository ppa:deadsnakes/ppa \
+  && apt-get update \
+  && apt-get install --yes --no-install-suggests --no-install-recommends \
+  python${PYTHON_VERSION} \
+  python${PYTHON_VERSION}-dev \
+  python${PYTHON_VERSION}-venv \
+  python${PYTHON_VERSION}-distutils \
+  && rm -rf /var/lib/apt/lists/* \
+  `#stage-pre-python`
+
+
+# Install wkhtmltopdf
+RUN wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox_${WKHTMLTOPDF_VERSION}-1.focal_amd64.deb \
+  && dpkg -i wkhtmltox_${WKHTMLTOPDF_VERSION}-1.focal_amd64.deb \
+  && rm wkhtmltox_${WKHTMLTOPDF_VERSION}-1.focal_amd64.deb \
+  `#stage-pre-wkhtmltopdf`
+
+
+# Install Fonts
+RUN git clone --progress --depth 1 https://github.com/frappe/fonts.git /tmp/fonts \
+  && rm -rf /etc/fonts && mv /tmp/fonts/etc_fonts /etc/fonts \
+  && rm -rf /usr/share/fonts && mv /tmp/fonts/usr_share_fonts /usr/share/fonts \
+  && rm -rf /tmp/fonts \
+  && fc-cache -fv \
+  `#stage-pre-fonts`
+
+
+# Set max_allowed_packet to 512 MB for mysqldump
+RUN echo "[mysqldump]\nmax_allowed_packet              = 512M" > /etc/mysql/conf.d/mysqldump.cnf
```

Comment:
Is this space between `max_allowed_packet` and `=` intentional ?

Reply:
That line is directly copied from prev dockerfile. Probably serves a purpose.